src/SFA.DAS.RoATPService.Application/Validators/IOrganisationValidator.cs

```diff
@@ -17,5 +17,7 @@ public interface IOrganisationValidator
         bool IsValidCharityNumber(string charityNumber);
         bool IsValidOrganisationType(OrganisationType organisationType);
         bool IsValidOrganisationTypeId(int organisationTypeId);
+
+        string DuplicateUkprnInAnotherOrganisation(long ukprn, Guid organisationId);
```

Comment:
Maybe return a model type for this, with is duplicate and the organisation name (if applicable) ?